src/Algebra/Module/Bundles.agda

```diff
@@ -92,6 +99,10 @@ record LeftModule (ring : Ring r ℓr) m ℓm : Set (r ⊔ ℓr ⊔ suc (m ⊔ 
     -ᴹ_ : Op₁ Carrierᴹ
     isLeftModule : IsLeftModule ring _≈ᴹ_ _+ᴹ_ 0ᴹ -ᴹ_ _*ₗ_
 
+  infix 4 _≉ᴹ_
+  _≉ᴹ_ : Rel Carrierᴹ _
+  a ≉ᴹ b = ¬ (a ≈ᴹ b)
```

Comment:
We shouldn't have to redefine this more than twice. For example here, we can import it from `leftSemimodule` below right?

Reply:
D'oh! I know better than to be unnecessarily redundant w/ code; sorry. :(
Fixed and arriving w/ next push.